@here/harp-datasource-protocol/lib/Expr.ts

```diff
@@ -626,7 +642,7 @@ export class MatchExpr extends Expr {
 }
 
 /**
- * @hidden
+ * A node representing a `case` expression.
```

Comment:
would it make sense to link to the syntax of this expression?

Reply:
all the expressions follow the same syntax and the link to the style expression is in the dcoumentation of the base Expr class.